configextractor.py

```diff
@@ -177,6 +177,14 @@ def subsection_builder(parent_section: ResultSection = None, fields: dict = {}):
         if field in fields:
             generic_section = ResultSection(f"Extracted {field.capitalize()}")
             field_data = fields[field]
+            if field == 'port':
```

Comment:
As a personal preference, I would like this specific port section-building code outside of the subsection_builder method, possibly right before the "other" section-building code, in order to maintain the "generic-ness" of the subsection_builder method.

Reply:
makes sense ! I'll move it.

Reply:
after discussion today, there will be no port tagging section.